sale_order_import_invoice2data/wizard/sale_order_import_pdf.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api, tools, _
+from openerp.exceptions import Warning as UserError
+from openerp.tools import float_compare
+from tempfile import mkstemp
+import base64
+import unicodecsv
+import os
+import re
+import pkg_resources
+import logging
+logger = logging.getLogger(__name__)
+
+try:
+    import unicodecsv
+except ImportError:
+    logger.debug('Cannot import unicodecsv')
+
+try:
+    from invoice2data.main import extract_data
+    from invoice2data.template import read_templates
+    from invoice2data.main import logger as loggeri2data
+except ImportError:
+    logger.debug('Cannot import invoice2data')
+
+class SaleOrderImport(models.TransientModel):
+    _inherit = 'sale.order.import'
+
+    # @api.model
+    # def fallback_parse_pdf_saleorder(self, file_data):
+    #     '''This method must be inherited by additionnal modules with
+    #     the same kind of logic as the account_bank_statement_import_*
+    #     modules'''
+    #     return self.invoice2data_parse_saleorder(file_data)
+    #
+    @api.model
+    def parse_pdf_order(self, file_data, detect_doc_type=False):
```

Comment:
I now made it that if the invoice parsing fails, it tries the the original XML-from-PDF parsing using `super()`